src/components/Extra.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <!-- --------------------------------------------------------------------------------------------- -->
     <!-- TabsBar -->
-    <TabsBar />
+    <TabsBar :isSimulationAreaClicked="isSimulationAreaClicked" @changeState="handleStateChange"/>
```

Comment:
```suggestion
    <TabsBar />
```
i think we do not need to change anything in this file can you remove all changes